src/tracer.js

```diff
@@ -253,6 +273,38 @@ export default class Tracer {
     }
 
     /**
+     * Reapplies sampling if the parent span isn't sampled
+     *
+     * @param parent the parent span context
+     * @param ctx the span context
+     * @param operationName the operation name for the child
+     * @param tags tags to be applied by the sampler
+     */
+    handleUpsampling(parent: SpanContext, ctx: SpanContext, operationName: string, tags: any) {
+        if (!this._upsampling.enabled) {
+            return
+        }
+
+        // Run upsampler on every span
+        if (this._upsampling.allowOnInnerSpans) {
+            if (this._sampler.isSampled(operationName, tags)) {
```

Comment:
I apologize, I understood https://github.com/uber/jaeger-client-node/pull/128#discussion_r125911128 to mean that you wanted me to add support for the configuration option. 
I would have understood you better if it was simply stated that there isn't a use case for the configuration and that we should always apply the decision on the root span to the inner spans.